code/unit_tests/_defines.dm

```diff
@@ -1 +1 @@
-#define MAX_UNIT_TEST_RUN_TIME 2 MINUTES
+#define MAX_UNIT_TEST_RUN_TIME 20 MINUTES
```

Comment:
👀 lol

Reply:
That was back when it was broken and would routinely time out. I think 10 minutes should probably be fine? It has to be at least the sum of the first two queue times, and maybe 20 seconds to a minute extra to process the queue (which is way faster now btw).